examples/README.md

```diff
@@ -74,16 +77,30 @@ As Vercel wants one single build command, we make use of the `build:vercel` npm
 
 **Note**: Vercel runs `npm install` in the root of the repository by default, so we do not need to worry about that.
 
-
 #### Environment Variables
-Once the application is configured and deployed (the first deploy will fail because of the missing environment variables, but we need to hit deploy before continuing as we need to configure a more complex set of variables which is not possible from the import screen), move to the application's Environment Variables (Settings > Environment Variables) and ensure to configure the following variables:
-
-| Name  | Type  | Value |
```

Comment:
You don't need to specify `Type` anymore as all variables are encrypted by default